vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/TimelineItemFactory.kt

```diff
@@ -130,11 +137,16 @@ class TimelineItemFactory @Inject constructor(
                     EventType.CALL_ANSWER -> callItemFactory.create(params)
                     // Crypto
                     EventType.ENCRYPTED -> {
-                        if (event.root.isRedacted()) {
+                        val relationContent = event.getRelationContent()
+                        when {
                             // Redacted event, let the MessageItemFactory handle it
-                            messageItemFactory.create(params)
-                        } else {
-                            encryptedItemFactory.create(params)
+                            event.root.isRedacted() -> messageItemFactory.create(params)
+                            relationContent?.type == RelationType.REFERENCE -> {
+                                // Hide the decryption error for VoiceBroadcast chunks
+                                val relatedEvent = relationContent.eventId?.let { session.getRoom(event.roomId)?.getTimelineEvent(it) }
```

Comment:
Maybe avoid requested a room everytime we need a timeline Event. Maybe make the room a lazy class member?